app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,59 @@
+*{
+  padding:0;
+  margin:0;
+}
+
+#container {
+  margin: 48px auto;
+  width: 960px;
+
+  section {
+    clear: both;
+    p {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3